services/api-server/tests/mocks/create_study_job.json

```diff
@@ -393,7 +393,7 @@
       "parent": null
      }
     },
-    "prjOwner": "bisgaard@itis.swiss",
+    "prjOwner": "dianaallen@example.com",
```

Comment:
Just to be sure, these changes are done by the cleanup script, and not manually, right? In case it's the latter, probably best to automate?

Reply:
yes it is done by the cleanup script
yes, it is best to automate e.g. attaching it in the pre-commit hook, but it needs some extra changes for this to happen

Reply:
I don't understand exactly why this is needed?